trulens_eval/trulens_eval/feedback/provider/openai.py

```diff
@@ -366,26 +366,25 @@ class AzureOpenAI(OpenAI):
     Has the same functionality as OpenAI out of the box feedback functions.
     """
 
-    def __init__(self, endpoint=None, deployment_name="gpt-35-turbo", **kwargs):
+    def __init__(self, deployment_name: str, endpoint=None, **kwargs):
```

Comment:
I dropped the default here since model deployment names in Azure can be anything. I can deploy `gpt-3.5-turbo` as `open-ai`.

Reply:
Is there a typical default that we can still use?

Reply:
I don't think so. There are no models deployed by default. You need to pick and choose which to deploy and choose the name at deployment time. The deployment name often doesn't match the model name due to Azure naming restrictions.